### PR TITLE
Fetch default-return-url from entity

### PR DESCRIPTION
**Pull request by benyamin-noori**
If returnUrl and document.referrer are both null, the back to home button should take us to content home by default. This link is provided by the siren entity.

### CODE REVIEW

File: d2l-sequence-viewer.html [hunk 1]
```diff
@@ -247,8 +247,9 @@
 			// if a return-url parameter was passed when creating the element
 			// then returnUrl should have that value. If not, returnUrl should
 			// be document.referrer.
-			_getReturnUrl() {
-				return this.returnUrl || document.referrer || '';
+			_getReturnUrl(entity) {
+				const defaultReturnUrl = entity && entity.getLinkByRel('default-return-url') || '';
```
Comment (jwong-d2l) [inline d2l-sequence-viewer.html]: is this a real thing, or are we adding it to the siren api?

Comment (benyamin-noori) [reply d2l-sequence-viewer.html]: https://git.dev.d2l/projects/CORE/repos/le/pull-requests/12892/overview

Comment (benyamin-noori) [reply d2l-sequence-viewer.html]: Done.

**Comment by jwong-d2l** [comment]
we should probably constantify the custom rels that we have if we make one more.

**Pull request merged**